go/ct/driver/cli/flags.go

```diff
@@ -62,12 +64,16 @@ var SeedFlag = &seedFlagType{
 	cli.Uint64Flag{
 		Name:    "seed",
 		Aliases: []string{"s"},
-		Usage:   "seed for the random number generator",
+		Usage:   "seed for the random number generator. If not provided, the seed is randomly generated",
 	},
 }
 
 func (f *seedFlagType) Fetch(context *cli.Context) uint64 {
-	return context.Uint64(f.Name)
+	if slices.Contains(context.FlagNames(), "seed") {
```

Comment:
there is a shortcut for this:
```suggestion
	if context.IsSet(f.Name) {
```